### PR TITLE
Improve tutorial 1

### PR DESCRIPTION
**Pull request by ValerieMichel**
This PR aims at reviewing and improving the tutorials of the user guide.

Test link: https://cyberbotics.com/doc/guide/tutorial-1-your-first-simulation-in-webots-20-minutes?version=fix-tutorials

### CODE REVIEW

File: docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md [hunk 1]
```diff
@@ -39,9 +39,9 @@ Click all the tick boxes, including the "Add a rectangle arena" which is not tic
 
 Webots displays a list of directories and files it just created.
 This corresponds to the [standard file hierarchy of a Webots project](the-standard-file-hierarchy-of-a-project.md).
-Click on the `Finish` button to close this window.
+Click on the `Done` button to close this window.
```
Comment (omichel) [inline docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md]: I indeed get a `Finish` button on Windows. Could that be different on macOS?

**Comment by ValerieMichel** [comment]
It is written Ctrl-C, Ctrl-V for copy and paste ; is it needed to write Cmd+C, Cmd+V for Mac?

The PROTO node is introduced with a link to the reference manual: A PROTO node defines a new node type in terms of built-in nodes or other PROTO nodes. I find this definition not very clear...

File: docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md (hunk 1 above)
Comment (ValerieMichel) [reply docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md]: Click on the `Finish` (Windows, Linux) or `Done` (macOS) button to close this window.

Comment (ValerieMichel) [reply docs/guide/tutorial-1-your-first-simulation-in-webots-20-minutes.md]:
```suggestion
Click on the `Finish` (Windows, Linux) or `Done` (macOS) button to close this window.
```

**Comment by ValerieMichel** [comment]
👍

**Comment by ValerieMichel** [comment]
👎

**Comment by ValerieMichel** [comment]
👍

**Comment by omichel** [comment]
:eyes:

File: docs/reference/proto-definition.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ### Interface
 
-The PROTO definition defines exactly what the PROTO does in terms of the built-in nodes or of the instances of other PROTO nodes.
+PROTO defines a new node name with custom fields that can be specified.
```
Comment (omichel) [inline docs/reference/proto-definition.md]:
```suggestion
A PROTO node is defined in a PROTO file.
A PROTO file ends with a `.proto` extension.
It lists the fields of the PROTO and defines how these fields impact the underlying object which is defined using base nodes and/or PROTO nodes.
```

**Comment by ValerieMichel** [comment]
👍

File: docs/reference/proto.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # PROTO
 
+The PROTO mechanism allows the user to extend the set of nodes by adding his own nodes. Thus, the user can build and re-use complex objects.
```
Comment (omichel) [inline docs/reference/proto.md]:
```suggestion
The PROTO mechanism allows users to extend the set of nodes by adding their own nodes.
Thus, users can build and re-use complex objects.
```

**Comment by ValerieMichel** [comment]
👍

**Pull request merged**